test/extended/builds/clone_git_protocol.go

```diff
@@ -0,0 +1,51 @@
+package builds
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-builds][Feature:Builds] clone repository using git:// protocol", func() {
+	var (
+		oc = exutil.NewCLI("build-pruning")
```

Comment:
nit - this is the prefix for the generated namespace.
```suggestion
		oc = exutil.NewCLI("build-clone-git-protocol")
```

Reply:
updated in next push